journal/original/12-chapter-10.adoc

```diff
@@ -95,10 +98,13 @@ and the Lord will be with you, (you shall see it), in whose name I send this amo
 Who am your friend and brother in the labour, tribulation,
 and patience of the gospel of peace, and a servant of the living God.
 
+[.signed-section-signature]
 Charles Marshall.
 
+[.signed-section-context-close]
 Written in the year 1677.
 
+[.blurb]
 Another Epistle to the flock of Christ Jesus: +++[+++exhorting to self-denial, stillness,
```

Comment:
Same here, needs the ===

Reply:
I'm thinking maybe I didn't look at the PDF on this chapter.....sorry about that!